test/e2e/restclient.go

```diff
@@ -0,0 +1,36 @@
+package e2e
+
+import (
+	"github.com/go-resty/resty/v2"
+	"github.com/kaleido-io/firefly/pkg/fftypes"
+)
+
+var (
+	urlGetNamespaces     = "/namespaces"
+	urlBroadcastDatatype = "/namespaces/default/broadcast/datatype"
+	urlGetData           = "/namespaces/default/data"
+)
+
+func GetNamespaces(client *resty.Client) (*resty.Response, error) {
+	return client.R().
+		SetResult(&[]fftypes.Namespace{}).
+		Get(urlGetNamespaces)
+}
+
+func BroadcastDatatype(client *resty.Client, name string) (*resty.Response, error) {
```

Comment:
It might make sense to pass in a `*fftypes.Datatype` here rather than just a name. That way different tests could define different data types.

Reply:
Yea, will probably change that in the future